portal-api/api-tests/v1/gef/facilities/facilities.put.api-test.js

```diff
@@ -525,6 +504,14 @@ describe(baseUrl, () => {
         expect(status).toBe(200);
       });
 
+      it('returns 200 when facilityEndDate & bank review date are both null', async () => {
+        const { body: facilityBody } = await as(aMaker).post({ dealId: mockApplication.body._id, type: FACILITY_TYPE.CASH, hasBeenIssued: false }).to(baseUrl);
+
+        const { status } = await as(aMaker).put({ facilityEndDate: null, bankReviewDate: null }).to(`${baseUrl}/${facilityBody.details._id}`);
+
+        expect(status).toBe(200);
```

Comment:
Do we want to use `HttpsStatusCode.OK` here? Not really fussed either way given it's a test file